README.md

```diff
@@ -210,6 +223,7 @@ There are 3 main classes in `node-resque`: Queue, Worker, and Scheduler
   - There's a special class called `multiWorker` in Node Resque which will run many workers at once for you (see below).
 - **Scheduler**: The scheduler can be thought of as the coordinator for Node Resque. It is primarily in charge of checking when jobs told to run later (with `queue.enqueueIn` or `queue.enqueueAt`) should be processed, but it performs some other jobs like checking for 'stuck' workers and general cluster cleanup.
   - You can (and should) run many instances of the scheduler class at once, but only one will be elected to be the 'leader', and actually do work.
+  - The 'delay' defined on a scheduled job does not specify when the job should be run, but rather when the job should be enqueued. This means that `node-resque` can not guarantee when a job is going to be executed, only when it will become _available_ for execution (added to a Queue).
```

Comment:
👍

Reply:
that's better! 

So finally, a "Scheduler" is a Worker, which poll... and check is the time configured is met. If it's the right time, it add the job on a specific queue.

Later, (soon) a worker will take the job.  I'm right?

---

If not, my confusion is about the similarity of a worker and a scheduler, there are not very different, listening same event, (more or less) poll job, check things...

Reply:
That is correct. A scheduler's primary responsibility it to move jobs from the Delayed Jobs Queues to the normal work Queues. It does not do any actual "work".

Reply:
> A scheduler's primary responsibility it to move jobs from the Delayed Jobs Queues to the normal work Queues. It does not do any actual "work".

This could be add to documentation, it's a great explanation